XamlControlsGallery/ControlPages/RichEditBoxPage.xaml.cs

```diff
@@ -34,14 +35,28 @@ public RichEditBoxPage()
         private void Menu_Opening(object sender, object e)
         {
             CommandBarFlyout myFlyout = sender as CommandBarFlyout;
-            if (myFlyout.Target == REBCustom)
+            if (myFlyout != null && myFlyout.Target == REBCustom)
             {
                 AppBarButton myButton = new AppBarButton
                 {
                     Command = new StandardUICommand(StandardUICommandKind.Share)
                 };
                 myFlyout.PrimaryCommands.Add(myButton);
             }
+            else
+            {
+                Mux::CommandBarFlyout muxFlyout = sender as Mux::CommandBarFlyout;

```

Comment:
Don't think that that works in C#.

Reply:
Agreed. Does this build?

Reply:
yes. it passed the compilation and no error in run time